client/layout/masterbar/logged-in.jsx

```diff
@@ -96,7 +96,7 @@ const MasterbarLoggedIn = React.createClass( {
 					className="masterbar__item-new"
 					tooltip={ this.translate( 'Create a New Post', { textOnly: true } ) }
 				>
-					{ this.translate( 'New Post' ) }
+					{ this.translate( 'Post' ) }
```

Comment:
I updated the translate call! 

> though I realized as I was commenting that I didn't know whether your intention was the verb or the noun :)

Indeed, that deserved clarification. Definitely the verb!